Engine/Engine/Assets/FragmentShader.frag

```diff
@@ -14,21 +9,4 @@ void main() {
 	gl_FragColor = vec4(uvFs, 0, 1);
```

Comment:
FYI, we might want to make an `out vec4 fragColor` instead using gl_FragColor, since it is depricated and, therefore, may not always work.